src/embedded_flash/mod.rs

```diff
@@ -19,6 +19,12 @@ mod helper;
 mod syscall;
 mod upgrade_storage;
 
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum UpgradeIdentifier {
+    A = 0,
+    B = 1,
```

Comment:
Do we actually need the `= 0` and `= 1`? We never serialize this or anything.

Reply:
I will be matched against its integer representation when parsing the [configure command input](https://github.com/kaczmarczyck/OpenSK/blob/96202434661fd57b9ea1220549a6fba3066e0f03/src/ctap/command.rs#L591). This is how the client stores a public key either for partition A or for partition B. Depending on the comment above, this might not be needed if we always configure both public keys right away.

Reply:
IMO rotating the signing keys shouldn't be done through a vendor command but part of the firmware update process: you push a new key set (both A and B), signed respectively by their previous ones.
The initial key set should be part of the bootloader installation.

Reply:
I just added my comment mentioning key rotation in parallel. Seems like this needs more discussion, as we might have to converge on how to bootstrap first. If we don't intend to bootstrap the public keys with a CTAP command, we can as well bake them into the app. I scheduled a discussion on that.